test/kubetest/tls.go

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
can you use any of https://github.com/openshift/kubernetes/blob/913d16a5d9fb76dfb617551d46b8c361a7128d3f/staging/src/k8s.io/client-go/util/cert/cert.go ?